adapter/serviceControl/serviceController.go

```diff
@@ -0,0 +1,197 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package serviceControl
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/pborman/uuid"
+	servicecontrol "google.golang.org/api/servicecontrol/v1"
```

Comment:
if we are going to have this package be `servicecontrol`, I recommend aliasing this import as something like `sc` or maybe `scapi`.